doc/docs/master/enterprise/auth/enterprise-server/setup.md

```diff
@@ -20,9 +20,58 @@ The setup of an Enterprise Server requires to:
 ## 1 - Deploy An Enterprise Server
 
 ### Single-cluster deployment
-Deploying a Pachyderm cluster with the embedded enterprise server does not require any specific action.
-Proceed as usual:
+Deploying and configuring a Pachyderm cluster with the embedded enterprise server can be done in one of two flavors.
+
+1. Provide all licensing and authentication configuration as a part of the Helm deployment.
+1. Install Pachyderm as usual with Helm, and use `pachctl` commands to set up licensing and authentication.
+
+#### To setup enterprise features as a part of the helm deployment
+Update your values.yaml with your enterprise license key, and auth configurations ([see example of values.yaml here](https://github.com/pachyderm/pachyderm/blob/master/etc/helm/examples/local-values.yaml)) or use our minimal example below.
+
+=== "values.yaml with activation of an enterprise license and authentication"
+
+	```yaml
+	pachd:
+		enterpriseLicenseKey: "<ENTEPRRISE-LICENSE-KEY>"
+		oauthClientId: "pachd"
+		oauthIssuer: "http://<PACHD-IP>:30658/"
+		oauthRedirectURI: "http://<PACHD-IP>:30657/authorization-code/callback"
+  		## if a secret is not provided, a secret will be autogenerated on install and stored in the k8s secret 'pachyderm-bootstrap-config.authConfig.clientSecret'
+  		oauthClientSecret: ""
+  		## if a secret is not provided, a secret will be autogenerated on install and stored in the k8s secret 'pachyderm-bootstrap-config.enterpriseSecret'
+  		enterpriseSecret: ""
+  		## if a token is not provided, a secret will be autogenerated on install and stored in the k8s secret 'pachyderm-bootstrap-config.rootToken'
+  		rootToken: ""
+  		## userAccessibleOauthIssuerHost is necessary in localhost settings or anytime the registered Issuer address isn't accessible outside the cluster
+  		# userAccessibleOauthIssuerHost: "localhost:30658"
```

Comment:
this section could technically be omitted since we handle these variables as optional on install - this would reduce the cognitive load going through this first pass.

On the other hand I think knowing these params exist is instructive

Reply:
I agree that these are instructive -  I would leave them there.